### PR TITLE
CTCP-5419: update page titles to include '-Departure declarations-'

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

**Comment by kenneth-rayner** [comment]
test me please

**Comment by kenneth-rayner** [comment]
Please test this

**Comment by kenneth-rayner** [comment]
Test this please

File: conf/application.conf [hunk 1]
```diff
@@ -30,7 +30,7 @@ play.filters.enabled += "play.filters.csp.CSPFilter"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
-play.additional.module = "config.TransitionModule"
+play.additional.module = "config.PostTransitionModule"
```
Comment (jackheslop96) [inline conf/application.conf:33]: Can this be reverted?

**Review by jackheslop96 — DISMISSED** [review]

**Review by JSquires91 — APPROVED** [review]

**Pull request merged**